hack/dev-setup-local

```diff
@@ -31,38 +77,38 @@ kubectl apply -f ${EXAMPLE_DIR}/70-secret-cloudprovider-local.yaml
 kubectl apply -f ${EXAMPLE_DIR}/80-secretbinding-cloudprovider-local.yaml
 
 kubectl config set-cluster gardener-dev \
-  --kubeconfig ${MINIKUBE_SEED_KUBECONFIG} \
-  --certificate-authority $HOME/.minikube/ca.crt \
-  --server https://${MINIKUBE_IP}:$API_SERVER_SECURE_PORT \
-  --embed-certs=true
+  --kubeconfig ${LOCAL_SEED_KUBECONFIG} \
+  ${TLS_VERIFICATION} \
+  --server https://${LOCAL_K8S_HOST_IP}:$API_SERVER_SECURE_PORT \
+  --embed-certs=false
```

Comment:
This should be `--embed-certs=true` for `minikube` and `--embed-certs=false` for `docker for desktop`